README.md

```diff
@@ -13,23 +13,61 @@ as of Mar 18, 2023.
 Jupiter is a research prototype. It contains known vulnerabilities and should not be used in production under any
 circumstances.
 
-## Getting Started
```

Comment:
Are we sure that we want to remove the getting started instructions?

Reply:
Yeah, they really shouldn't have been there in the first place. Jupiter is a library - it just happened to have a `main` function that I used for testing at one point.

Reply:
Makes sense!